locales/package.json

```diff
@@ -1,12 +1,9 @@
 {
   "name": "locales",
-  "version": "0.0.1",
+  "version": "1.0.0",
   "description": "Locales for the integreat frontend projects",
   "main": "index.js",
-  "author": "Maximilian Ammann <max@maxammann.org>",
-  "license": "MIT",
-  "private": false,
```

Comment:
probably should remain private

Reply:
It was not private before. However, made all packages private now.